demisto_sdk/commands/download/downloader.py

```diff
@@ -95,647 +87,851 @@
+                )[0]
+
+            if not isinstance(api_response, dict):
+                # If there are API issues, it might return HTML, which is returned as a string.
+                raise ApiException(
+                    f"Unexpected response from server:\n" f"'{api_response}'."
+                )
+
+            downloaded_playbooks.append(api_response)
+
+        logger.debug(
+            f"'{len(downloaded_playbooks)}' system playbooks were downloaded successfully."
+        )
+        return downloaded_playbooks
+
+    def generate_content_file_name(
+        self, content_item: dict, content_item_type: str
+    ) -> str:
+        item_name: str = content_item.get("name") or content_item["id"]
+        result = (
+            item_name.replace("/", "_").replace(" ", "_")
+            + ITEM_TYPE_TO_PREFIX[content_item_type]
         )
 
-    def fetch_system_content(self):
+        # Remove duplicate underscores
+        return re.sub(r"_{2,}", "_", result)
+
+    def fetch_system_content(self, content_item_type: str) -> dict[str, dict]:
         """
-        Fetches the system content from Demisto into a temporary dir.
-        :return: True if fetched successfully, False otherwise
+        Fetch system content from the server.
+
+        Returns:
+            dict[str, dict]: A dictionary mapping content item's file names, to dictionaries containing metadata
+                and content of the item.
         """
-        try:
-            system_items_list: Union
-            endpoint, req_type, req_body = self.build_req_params()
+        endpoint, req_type, req_body = self.build_req_params(
+            content_item_type=content_item_type
+        )
+        downloaded_items: list[dict]
 
-            if self.system_item_type == "Automation":
-                system_items_list = self.get_system_automation(req_type)
+        if content_item_type == "Automation":
+            downloaded_items = self.get_system_automation(
+                content_items=self.input_files
+            )
+
+        elif content_item_type == "Playbook":
+            downloaded_items = self.get_system_playbook(content_items=self.input_files)
 
-            elif self.system_item_type == "Playbook":
-                system_items_list = self.get_system_playbook(req_type)
+        else:
+            logger.info("Fetching system items...")
+            api_response = demisto_client.generic_request_func(
+                self.client,
+                endpoint,
+                req_type,
+                body=req_body,
+                response_type="object",
+            )[0]
+
+            if content_item_type in ("Classifier", "Mapper"):
+                if classifiers_data := api_response.get("classifiers"):
+                    downloaded_items = classifiers_data
+
+                else:
+                    logger.warning(
+                        "Could not find expected 'classifiers' key in API response."
+                    )
```

Comment:
```suggestion
                    logger.debug(
                        "Could not find expected 'classifiers' key in API response."
                    )
```

Reply:
think about a user that sees this log and has no knowledge in apis, it doesn't say anything to him